app/src/main/java/com/codepath/earthquakemonitor/JsonHttpResponseHandler/EarthquakesJsonHttpResponseHandler.java

```diff
@@ -0,0 +1,70 @@
+package com.codepath.earthquakemonitor.JsonHttpResponseHandler;
+
+import android.util.Log;
+
+import com.codepath.earthquakemonitor.models.Earthquake;
+import com.loopj.android.http.JsonHttpResponseHandler;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+import cz.msebera.android.httpclient.Header;
+
+/**
+ * Created by emilie on 10/9/17.
+ */
+
+public class EarthquakesJsonHttpResponseHandler extends JsonHttpResponseHandler{
+
+    private final String TAG = "EQJsonHandlerTAG";
+
+    @Override
+    public void onSuccess(int statusCode, Header[] headers, JSONObject object) {
+        //todo here modifying what to do with the answer
+        ArrayList<Earthquake> earthquakes = new ArrayList<Earthquake>();
+        try {
```

Comment:
In the future when we have the Map Activity we will move the arrayList there I let it there for the moment so we can test it.